### PR TITLE
Task jd/j160 add genre info to podcast page

### PR DESCRIPTION
**Pull request by JuliaDzyuba**

### CODE REVIEW

File: backend/src/data/models/podcast/podcast.model.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { TableName, PodcastDTOKey, PodcastType } from '~/common/enums/enums';
 import { Abstract } from '../abstract/abstract.model';
 import { Image } from '~/data/models/image/image.model';
 import { User } from '~/data/models/user/user.model';
+import { Genre } from '../genre/genre.model';
```
Comment (what1s1ove) [inline backend/src/data/models/podcast/podcast.model.ts]:
```suggestion
import { Genre } from '~/data/models/genre/genre.model';
```

File: backend/src/data/models/podcast/podcast.model.ts [hunk 2]
```diff
@@ -25,6 +26,8 @@ class Podcast extends Abstract {
 
   [PodcastDTOKey.USER]: User;
 
+  [PodcastDTOKey.GENRE]: Genre;
```
Comment (what1s1ove) [inline backend/src/data/models/podcast/podcast.model.ts]:
```suggestion
  [PodcastDTOKey.GENRE]: Genre | null;
```

File: frontend/src/components/podcast/podcast.tsx [hunk 3]
```diff
@@ -64,6 +64,10 @@ const Podcast: React.FC = () => {
                   <div className={`${styles.infoName} ${styles.period}`}>Рeriodicity</div>
                   <p className={styles.infoInner}>Once a month</p>
                 </li>
+                <li className={styles.infoItem}>
+                  <div className={`${styles.infoName} ${styles.genre}`}>Genre</div>
```
Comment (what1s1ove) [inline frontend/src/components/podcast/podcast.tsx]:
```suggestion
                  <div className={getAllowedClasses(styles.infoName, styles.genre)}>Genre</div>
```
always use it when you need more than 1 class

File: shared/src/common/types/podcast/podcast.type.ts [hunk 4]
```diff
@@ -16,6 +17,7 @@ type Podcast = {
   type: PodcastType;
   genreId: number | null;
   user: User;
+  genre: Genre;
```
Comment (what1s1ove) [inline shared/src/common/types/podcast/podcast.type.ts]:
```suggestion
  genre: Genre | null;
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by Denidzio — DISMISSED** [review]

File: frontend/src/components/podcast/podcast.tsx [hunk 5]
```diff
@@ -5,6 +5,7 @@ import { Link, Loader, ImageWrapper } from 'components/common/common';
 import { EpisodeTable } from './components/components';
 import { PageParams } from './common/types/types';
 import styles from './styles.module.scss';
+import { getAllowedClasses } from 'helpers/dom/dom';
```
Comment (what1s1ove) [inline frontend/src/components/podcast/podcast.tsx]:
```suggestion
import { getAllowedClasses } from 'helpers/helpers';
```
keep import order

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/image-preview-control/image-preview-control.tsx [hunk 6]
```diff
@@ -9,8 +9,8 @@ import {
   Control,
 } from 'react-hook-form';
 import { ErrorMessage } from '@hookform/error-message';
-import { getFileExtensions } from 'helpers/helpers';
-import { getAllowedClasses } from 'helpers/dom/dom';
+import { getFileExtensions, getAllowedClasses } from 'helpers/helpers';
+// import { getAllowedClasses } from 'helpers/dom/dom';
```
Comment (what1s1ove) [inline frontend/src/components/common/image-preview-control/image-preview-control.tsx]:
```suggestion
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**